.github/workflows/python-test.yml

```diff
@@ -4,6 +4,7 @@ jobs:
   python-test:
     runs-on: ubuntu-latest
```

Comment:
Can we broaden our testing to windows and macos in one version as well? Just to make sure that the behavior is the same (particularly for Windows). :sweat_smile: 
This should just be a matter of redefining the matrix here and the `runs-on`.

Reply:
Just did it.